.travis.yml

```diff
@@ -2,13 +2,14 @@ language: python
 python:
   - "2.6"
   - "2.7"
-  - "3.2"
   - "3.3"
   - "3.4"
   - "3.5"
+  - "3.6"
```

Comment:
Maybe add nigthly version (and allow failure) like:
```
    - 'nightly'
jobs:
  allow_failures:
    - python: "nightly"
```

Reply:
@kryskool done!

Reply:
@4383 Thanks